.buildkite/pipeline.yml

```diff
@@ -7,10 +7,10 @@ steps:
   - name: "Upgrade Test"
     command:
     - apk add g++ make bash gcompat curl mysql mysql-client libc6-compat
-    - wget https://golang.org/dl/go1.20.2.linux-amd64.tar.gz
-    - tar -C /usr/local -xzf go1.20.2.linux-amd64.tar.gz
+    - wget https://golang.org/dl/go1.21.10.linux-amd64.tar.gz
+    - tar -C /usr/local -xzf go1.21.10.linux-amd64.tar.gz
```

Comment:
I am guessing this is `1.21.10` instead of `1.21.11` (what we use on `release-17.0`) since the latest `v17.0.x` tag is on `1.21.10`.

https://github.com/vitessio/vitess/blob/v17.0.7/go.mod

Reply:
Exactly